docs/decisions/.template

```diff
@@ -25,3 +25,5 @@ What decision did you go forward with, and how does it map to the above decision
 ## Appendix (OPTIONAL)
 
 Add any links here that are relevant for understanding your proposal or its background.
+
+**Be sure to read the information about this in [CONTRIBUTING](CONTRIBUTING.md##Requests-for-comment)**
```

Comment:
It might be better to make this a link similar to the suggestion.
```suggestion
**Be sure to read the information about this in [CONTRIBUTING](https://github.com/CDCgov/dibbs-ecr-refiner/blob/main/CONTRIBUTING.md##Request-for-comment)**
```